test/load-order-3.html

```diff
@@ -32,7 +32,7 @@
 <body>
   <section class="main">
     <h1>A page with large images (delays after DOMContentLoaded).</h1>
-    <img src="http://lorempixel.com/1600/1600/" alt="Large image">
+    <img src="https://loremflickr.com/1600/1600/" alt="Large image">
```

Comment:
I don't get a 1600x1600 image, though, but I don't have a replacement in mind. Feel free to adapt this later :)

Reply:
Gotcha. This is definitely an improvement.

Reply:
How about https://picsum.photos/1600 ? 😊